src/ui/server/controllers/insurance/policy/save-data/policy/index.ts

```diff
@@ -1,7 +1,13 @@
 import api from '../../../../../api';
 import getDataToSave from '../../../../../helpers/get-data-to-save';
+import stripEmptyFormFields from '../../../../../helpers/strip-empty-form-fields';
 import { sanitiseData } from '../../../../../helpers/sanitise-data';
 import { Application, RequestBody } from '../../../../../../types';
+import POLICY_FIELD_IDS from '../../../../../constants/field-ids/insurance/policy';
+
+const { CREDIT_PERIOD_WITH_BUYER } = POLICY_FIELD_IDS;
+
+export const nullOrEmptyStringFields = [CREDIT_PERIOD_WITH_BUYER];
```

Comment:
Should we capitilise this? Since other local variables in controllers are capitilised, e.g `TEMPLATE, FIELD_IDS, PAGE_CONTENT_STRINGS`

Also can we add some documentation?